images-list

```diff
@@ -223,10 +229,15 @@ quay.io/kiali/kiali rancher/kiali-kiali v1.24.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.28.0
 quay.io/kiali/kiali rancher/kiali-kiali v1.29.0
 quay.io/kiali/kiali rancher/mirrored-kiali-kiali v1.29.0
+quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.10.7
```

Comment:
These are listed up above on line 128

Reply:
@cbron since this is now coming from quay.io, shouldn't we keep it here to maintain alphabetical order?

Reply:
Oh, yea ok